src/functions/op.js

```diff
@@ -4,8 +4,9 @@ import defineFunction, {ordargument} from "../defineFunction";
 import buildCommon from "../buildCommon";
 import {SymbolNode} from "../domTree";
 import * as mathMLTree from "../mathMLTree";
-import utils from "../utils";
+import utils from "../utils/utils";
 import Style from "../Style";
+import {assembleSupSub} from "../utils/assembleSupSub";
```

Comment:
I was thinking that `assembleSupSub` would go in `src/functions/utils/assembleSupSub.js`.  That would avoid unrelated changes to files importing `src/utils.js`.  Sorry for not being clearer in my original comment.  The rest of the PR looks good.  I'll push a commit to move things around and then approve this.